maintainers/maintainer-list.nix

```diff
@@ -2587,6 +2587,12 @@
     githubId = 91113;
     name = "David Kleuker";
   };
+  davidarmstronglewis = {
+    email = "davidlewis@mac.com";
+    github = "davidarmstronglewis";
+    githubId = 6754950;
+    name = "David Armstrong Lewis";
+  };
```

Comment:
should be a separate commit

Reply:
Please make this into a separate commit:

```
maintainers: add <name>
```

Reply:
Done in #145441